src/anemoi/inference/metadata.py

```diff
@@ -488,34 +488,65 @@ def area(self) -> Optional[str]:
         """Return the area information."""
         return self._data_request.get("area")
 
-    def variables_from_input(self, *, include_forcings: bool) -> list:
+    def select_variables(self, *, include, exclude) -> list:
         """Get variables from input.
 
         Parameters
         ----------
-        include_forcings : bool
-            Whether to include forcings.
+        include: List[str]
+            Categories to include.
+        exclude: List[str]
+            Categories to exclude.
 
         Returns
         -------
         list
             The list of variables.
         """
+
+        CATEGORIES = {"computed", "forcing", "diagnostic", "prognostic", "constant", "accumulation"}
```

Comment:
Can you print the list of possible categories somewhere in the retrieve CLI ? Not just in an error message if you get it wrong.

Reply:
Done